modelci/hub/deployer/serving.py

```diff
@@ -1,6 +1,4 @@
 import argparse
```

Comment:
![image](https://user-images.githubusercontent.com/5894780/86141874-2197c500-bb25-11ea-8bfd-c42f79dab67d.png)
why delete this? It seems good
```
from .serving import serve as dispatcher

__all__ = ['dispatcher']
```

Reply:
this will raise the circular reference issue.
```
serving deps: manager
manger deps: profiler
profiler deps: serving
```
Cannot put in `__init__.py`. Could you suppose a better structure?

Reply:
Add a todo, we will fix this up later.

Reply:
Done